test/shared_test/lib_battery_test.h

```diff
@@ -257,8 +262,8 @@ class lib_battery_test : public ::testing::Test
         dtHour = 1.0;
 
         capacityModel = new capacity_lithium_ion_t(q, SOC_init, SOC_max, SOC_min, dtHour);
-        voltageModel = new voltage_dynamic_t(n_series, n_strings, Vnom_default, Vfull, Vexp, Vnom, Qfull, Qexp, Qnom,
-                                             C_rate, resistance, dtHour);
+        voltageModel = new voltage_dynamic_t(n_series, n_strings, Vnom_default, Vfull, Vexp, Vnom, Qfull, Qexp, Qnom, Vcut, 
+                                             C_rate, resistance, dtHour );
```

Comment:
With the code changes there are always going to be issues when the q value used to initialize the capacity (q = 1000) is not tied to the cell capacity and number of strings (Qfull = 2.25, n_strings = 9).